src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -27,4 +27,16 @@ public Integer getLength() {
     public String getColor() {
         return color;
     }
+
+    public boolean equals(Rectangle ob) {
+        return (width.intValue() == ob.getWidth()) && (length.intValue() == ob.getLength());
+
+    }
+
+    public int hashCode() {
+        int result = 31;
```

Comment:
@kenu21 Ok. This is fine. Did you know why here is using 31?

Reply:
> @kenu21 Ok. This is fine. Did you know why here is using 31?
I watched video about hashCode to our homework. It explained there.